packages/node/src/plugins/storage/StorageEventListener.ts

```diff
@@ -11,8 +11,8 @@ export class StorageEventListener {
     private readonly clusterId: EthereumAddress
     private readonly streamrClient: StreamrClient
     private readonly onEvent: (stream: Stream, type: 'added' | 'removed', block: number) => void
-    private readonly onAddToStorageNode: (event: StorageNodeAssignmentEvent) => void
-    private readonly onRemoveFromStorageNode: (event: StorageNodeAssignmentEvent) => void
+    private readonly onAddToStorageNode: (event: StorageNodeAssignmentEvent) => unknown
+    private readonly onRemoveFromStorageNode: (event: StorageNodeAssignmentEvent) => unknown
```

Comment:
Why are these changes needed?

Reply:
So that the implementation can be async method (can return e.g. `Promise<void>`). Alternatively we could use `setImmedate`/`executeSafePromise` wrapper in the `onAddToStorageNode` implementation.

Reply:
Reverted in 20ab88664